test/JDBC/expected/BABEL_OBJECT_ID-vu-verify.out

```diff
@@ -0,0 +1,218 @@
+-- test table, procedure, function, views
+SELECT OBJECT_NAME(OBJECT_ID('babel_object_id_t1'))
```

Comment:
We should add few negative test cases, for NULL input and also for excessively long name such that it gets truncated.